src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $color-grey;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    color: $color-main-accent;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__product-code,
+  &__review-value {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    color: $color-secondary;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-value {
+      color: $color-main-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      color: $color-secondary;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-value {
+      color: $color-main-accent;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__price-value,
+  &__buy-btn {
+    font-weight: 700;
+  }
+
+  &__buy-btn {
+    box-sizing: border-box;
+    text-decoration: none;
+    width: 166px;
+    height: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    color: #fff;
+    background-color: $color-accent-blue;
+    border: 1px solid transparent;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+  }
+
+  &__buy-btn:hover {
+    border: 1px solid $color-accent-blue;
```

Comment:
as for the previous hw's you should use the change only of border-color

Reply:
agree